Formula/berkeley-db.rb

```diff
@@ -13,10 +13,10 @@ class BerkeleyDb < Formula
     sha256 "8f5a87bc3336e01f8528ae8aaae24c83e8fa10e8f01ee65e8cac4af7d0786bf1" => :mountain_lion
   end
 
-  option "with-java", "Compile with Java support."
-  option "with-sql", "Compile with SQL support."
+  option "without-sql", "Build without SQL support."
```

Comment:
Do you mean we should convert "without-sql" to "with-sql" OR `depends_on "sql" => :build` ? I guess the latter but want to make sure